src/Microsoft.Framework.SecurityHelper.Sources/project.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.0.0-*",
+    "shared": "*.cs",
+    "dependencies": {
+    },
+    "frameworks": {
+        "net45": { },
+        "dnx451": { },
+        "dnxcore50": {
+            "dependencies": {
+                "System.Runtime": "4.0.20-*"
```

Comment:
We don't have a claims dependency in Security today either, but maybe someone else brings it in?